doc/api/animation_api.rst

```diff
@@ -97,6 +97,11 @@ this hopefully minimalist example gives a sense of how ``init_func``
 and ``func`` are used inside of `FuncAnimation` and the theory of how
 'blitting' works.
 
+.. note::
+
+    The zorder of artists is not taken into account when 'blitting'
+    as the artists are always 'blit' on top.
```

Comment:
```suggestion
    as the blitted artists are always drawn on top.
```

Reply:
Makes sense. Should I add (single...) quotation marks around 'blitted' here or remove them elsewhere?